design/sequence/scheduler-low.puml

```diff
@@ -0,0 +1,108 @@
+@startuml
+
+box "server"
+    participant ":Database" as db
+    participant ":Scheduler" as s
+    participant "j:Job" as j
+    participant ":Dispatcher" as d
+end box
+
+s++
+s -> db++: get_jobs()
+db --> s--: jobs: list[Job]
+
+s -> s++: sorted(jobs)
+note right of s
+    Sorts jobs list by effective priority.
+end note
+s --> s--: jobs
+
+s -> db++: get_available_workers()
+note right of db
+    Return workers with free resources.
+end note
+db --> s--: available_wcs: list[WorkMachine]
+
+loop j in jobs
+    
+    s->s++ : preemption_threshold()
+    s-->s-- : preemption : float
+    
+    alt j.get_eff_priority > preemption
+        s -> db++ : lowest_priority_wm()
+        note right of s
+            Returns WorkMachine
+            whose jobs have the
+            lowest summed priority.
```

Comment:
Should be the work machine where the maximum priority of the Jobs that would need to be paused is lowest.